Ginger/GingerCore/Drivers/Appium/SeleniumAppiumDriver.cs

```diff
@@ -735,35 +797,38 @@ private void GenElementHandler(ActGenElement act)
-                                        ((AndroidDriver<AppiumWebElement>)Driver).PressKeyCode(22);
-                                        //Driver.KeyEvent(67);//"KEYCODE_DEL"- delete 1 character
-                                        ((AndroidDriver<AppiumWebElement>)Driver).PressKeyCode(67);
-                                    }
-                                }
-                            }
-                            catch (Exception ex) { Reporter.ToLog(eLogLevel.ERROR, $"Method - {MethodBase.GetCurrentMethod().Name}, Error - {ex.Message}", ex); }
+                            //e.Clear();
+                            ////make sure value was cleared- trying to handle clear issue in WebViews
+                            //try
+                            //{
+                            //    //TODO: Need to add a flag in the action for this case, as sometimes the value is clear but show text under like 'Searc, or say "OK Google".
+                            //    //Wasting time when not needed
+                            //    string elemntContent = e.Text; //.GetAttribute("name");
+                            //    if (string.IsNullOrEmpty(elemntContent) == false)
+                            //    {
+                            //        for (int indx = 1; indx <= elemntContent.Length; indx++)
+                            //        {
+                            //            //Driver.KeyEvent(22);//"KEYCODE_DPAD_RIGHT"- move marker to right
+                            //            ((AndroidDriver<AppiumWebElement>)Driver).PressKeyCode(22);
+                            //            //Driver.KeyEvent(67);//"KEYCODE_DEL"- delete 1 character
+                            //            ((AndroidDriver<AppiumWebElement>)Driver).PressKeyCode(67);
+                            //        }
+                            //    }
+                            //}
+                            //catch (Exception ex) { Reporter.ToLog(eLogLevel.ERROR, $"Method - {MethodBase.GetCurrentMethod().Name}, Error - {ex.Message}", ex); }
                             switch (DriverPlatformType)
                             {
                                 case SeleniumAppiumDriver.eSeleniumPlatformType.Android:
+                                    e.Clear();
                                     e.SendKeys(act.GetInputParamCalculatedValue("Value"));                                    
                                     break;
                                 case SeleniumAppiumDriver.eSeleniumPlatformType.iOS:
-                                    ((IOSElement)e).SetImmediateValue(act.GetInputParamCalculatedValue("Value"));
+                                    //e.Clear();
+                                    e.SendKeys(act.GetInputParamCalculatedValue("Value"));
+                                    //((IOSElement)e).SetImmediateValue(act.GetInputParamCalculatedValue("Value"));
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6113827)